dist/skeleton/skeleton.css

```diff
@@ -0,0 +1,234 @@
+  --skeleton-tile-width-min: 121px;
+  --skeleton-width: 100%;
+}
+.skeleton-avatar {
+  background: var(--color-loading-fill);
+  border-radius: 50%;
+  cursor: progress;
+  height: var(--skeleton-height);
+  min-height: 12px;
+  width: var(--skeleton-height);
+}
+.skeleton-button {
+  background: var(--color-loading-fill);
+  border-radius: 20px;
+  cursor: progress;
+  height: 40px;
+  min-height: 12px;
+  width: var(--skeleton-width);
+}
+.skeleton-button--small {
+  background: var(--color-loading-fill);
+  border-radius: 16px;
+  cursor: progress;
+  height: 32px;
+  min-height: 12px;
+  width: var(--skeleton-width);
+}
+.skeleton-button--large {
+  background: var(--color-loading-fill);
+  border-radius: 24px;
+  cursor: progress;
+  height: 48px;
+  min-height: 12px;
+  width: var(--skeleton-width);
+}
+.skeleton-text,
+.skeleton-text--small {
+  background: var(--color-loading-fill);
+  border-radius: 3px;
+  cursor: progress;
+  height: 16px;
+  min-height: 12px;
+  width: var(--skeleton-width);
+}
+.skeleton-text::after,
+.skeleton-text--small::after {
+  height: 16px;
+  margin-top: calc(16px + var(--spacing-100));
+}
+.skeleton-text--large {
+  background: var(--color-loading-fill);
+  border-radius: 3px;
+  cursor: progress;
+  height: 24px;
+  min-height: 12px;
+  width: var(--skeleton-width);
+}
+.skeleton-text--large::after {
+  height: 24px;
+  margin-top: calc(24px + var(--spacing-100));
+}
+.skeleton-text--multiline {
+  margin-bottom: var(--spacing-300);
+  position: relative;
+  width: calc(100% - var(--spacing-300));
+}
+.skeleton-text--multiline::after {
+  background: inherit;
+  content: "";
+  position: absolute;
+  width: calc(100% - var(--spacing-700));
+}
+.skeleton-text--large.skeleton-text--multiline {
+  margin-bottom: var(--spacing-400);
+}
+.skeleton-form {
+  background: var(--color-loading-fill);
+  border-radius: 8px;
+  cursor: progress;
+  height: 48px;
+  min-height: 12px;
+  width: var(--skeleton-width);
+}
+.skeleton-image {
+  background: var(--color-loading-fill);
+  border-radius: 8px;
+  cursor: progress;
+  height: 100%;
+  min-height: 12px;
+  width: var(--skeleton-width);
+  container-name: skeleton-image;
+  container-type: inline-size;
+  max-width: initial;
+}
+.skeleton-image::after {
+  bottom: 0;
+  height: 100%;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 100%;
+}
+span.skeleton-avatar,
+span.skeleton-button,
+span.skeleton-text,
+span.skeleton-image,
+span.skeleton-form {
+  display: inline-block;
+}
+span.skeleton-avatar + span,
+span.skeleton-button + span,
+span.skeleton-text + span,
+span.skeleton-image + span,
+span.skeleton-form + span {
+  -webkit-margin-start: var(--spacing-100);
+          margin-inline-start: var(--spacing-100);
+}
+div.skeleton-avatar,
+div.skeleton-button,
+div.skeleton-text,
+div.skeleton-image,
+div.skeleton-form {
+  -webkit-margin-before: var(--spacing-150);
+          margin-block-start: var(--spacing-150);
+}
+.skeleton-tile {
```

Comment:
Maybe should be called item-tile instead of tile.

Reply:
If that's the case, I don't think it should be in skin unless we also add item tiles to skin

Reply:
We might not even need it, since adding the sensible vertical margin default to text block and image, those should give us almost everything we need to easily construct an item tile...

Reply:
I'm glad we came to the [same conclusion](https://github.com/eBay/skin/pull/2177#discussion_r1399632241).

Reply:
Potentially, just got to double check the container class isn't providing any useful styles. If not, yes we can just show this an example of how to construct a skeleton of an item tile.